psi4/src/psi4/libmints/rel_potential.cc

```diff
@@ -31,71 +31,55 @@
 #include "psi4/libmints/integral.h"
 #include "psi4/libmints/matrix.h"
 #include "psi4/libmints/molecule.h"
+#include "psi4/libpsi4util/PsiOutStream.h"
 
-#include <libint2/shell.h>
 #include <libint2/engine.h>
 
-#include <algorithm>
-
 #define MAX(a, b) ((a) > (b) ? (a) : (b))
 
 #define RELVDEBUG 0
 
 ;
 using namespace psi;
 
-// Initialize potential_recur_ to +1 basis set angular momentum
 RelPotentialInt::RelPotentialInt(std::vector<SphericalTransform>& st, std::shared_ptr<BasisSet> bs1,
                                  std::shared_ptr<BasisSet> bs2, int deriv)
     : OneBodyAOInt(st, bs1, bs2, deriv) {
     if (deriv > 0) {
         throw PSIEXCEPTION("RelPotentialInt: deriv > 0 is not supported.");
     }
 
+    if (bs1 != bs2) {
```

Comment:
Thank you for pointing this out. I am wondering if we should throw and exception here since this is serious stuff.

Reply:
I just copy-pasted from `potential.cc`.

Reply:
Thanks for checking. All looks good to me!